### DISCUSSION TITLE
GitHub Sponsors Updates

### DISCUSSION THREAD

**Discussion by tuves** [community/community#51969] [Announcements]
Today, GitHub Sponsors announced new features: invoicing and a sponsorship dashboard are now generally available to all organizations making sponsorships as well as a bulk sponsorship feature available to all users, which allows for checking out with multiple sponsorships at once. Organization-funded sponsorships are nearly 15x more valuable to maintainers and these changes make it easier for them to sponsor at scale. Read more about all the updates on our [blog](https://github.blog/2023-04-04-whats-new-with-github-sponsors/).

**Comment by chardso** [comment]
Ooh what a great achievement big ups GitHub

**Comment by DIMA-1000** [comment]
why exactly 15 times?? not 8 or 17

**Comment by Souvikparua** [comment]
ooh

**Comment by oldcityws** [comment]
Is there an ETA on decision re: application for eligibility to accept sponsors?  I've been waiting some time now.  Not sure if I am in the middle of a huge backlog, but I am hoping to be able to start earning money by way of sponsorship.  Feedback / assistance will be greatly appreciated, thank you for any help or direction that you can give!

**Reply by DHayes10** [reply]
Hello @oldcityws my apologies for the delay. It looks like your profile was reviewed on 2/13/23 but was lacking a bit of public activity. I will approve this profile. Thank you for your contribution to open source! If you have any questions or concerns please feel free to reach out.

**Comment by rallaserdener** [comment]
EEEWEW

**Reply by Souvikparua** [reply]
Good

**Comment by phongsak52050com** [comment]
ในวันที่ พฤ. 6 เม.ย. 2023 21:53 น. chardso ***@***.***>
เขียนว่า:

**Comment by Mahmoud4g** [comment]
Thanks a lot for the great service and great information that you gave. Thanks to all

**Comment by Ameralameri** [comment]
Good

**Comment by blakeNaccarato** [comment]
The bulk sponsorship explorer is very cool! In addition to filtering down to "direct dependencies only", can a filter be created to filter down to "my repositories only" as well? Dependencies in my forks of contributions to big projects are cluttering the explore page.

**Reply by emmaviolet** [reply]
Thanks for this feedback @blakeNaccarato! So just to be clear, you want to exclude dependencies of any forked repositories you own? Seems reasonable and doable 👍

**Reply by blakeNaccarato** [reply]
Yep, it would be another interesting slice of the data. An "escape hatch" of sorts could also facilitate arbitrary analysis like this, in case you're trying to keep the Explore menu simpler to facilitate quick sponsoring. Like if the content that populates the explore sponsors page could be exported to JSON, with granular information such as the user's repositories that the dep appears in and so on. That opens the door to users writing tools to operate on that JSON, filtering in arbitrary ways, and producing a custom CSV for bulk sponsoring.

**Comment by Ameralameri** [comment]
Good

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
